doc/userguide/rules/ssh-keywords.rst

```diff
@@ -19,9 +19,9 @@ The example above matches on SSH connections with SSH version 2.
 
 ``ssh.proto`` can be used as ``fast_pattern``.
 
-``ssh.proto`` replaces the previous keyword name: ``ssh_proto``. You may continue
-to use the previous name, but it's recommended that rules be converted to use
-the new name.
+``ssh.proto`` replaces the previous keyword names: ``ssh_proto`` & 
+``ssh.protoversion``. You may continue to use the previous name, but 
```

Comment:
It might be good to distinguish between ```ssh_proto``` and ```ssh.protoversion``` as they have quite different mechanics. ```ssh_proto``` is just the old name for ```ssh.proto```, but operates the same way as sticky buffers. ```ssh.protoversion``` is quite different.